doc/usage/installation.rst

```diff
@@ -187,3 +187,45 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
 __ https://github.com/sphinx-doc/sphinx
 __ https://github.com/sphinx-doc/sphinx/archive/master.tar.gz
 __ https://github.com/sphinx-doc/sphinx/archive/master.zip
+
+
+docker-sphinx
```

Comment:
Docker is a kind of software platform. So it is better to use "Docker" simply to me.

Reply:
Thanks. I changed the title.